packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -1299,6 +1459,159 @@ void main() {
     await tester.tap(find.text('CANCEL'));
     await tester.pumpAndSettle();
   });
+
+
+  group('Material 2', () {
+    // Tests that are only relevant for Material 2. Once ThemeData.useMaterial3
```

Comment:
We will not remove the tests until the support for M2 is removed. When M3 is the default, the support for M2 will continue. At some point after M3 becomes the default we'll deprecate the M2 support. Once another ~year has gone by (per the deprecation policy) we'll remove the M2 support.

Reply:
This comment is from existing Material 2 group tests. I think we update all matches at once in the framework for consistency in a separate PR. WDYT?

Reply:
SGTM